recipe/meta.yaml

```diff
@@ -1,52 +1,77 @@
-{% set name = "scs" %}
 {% set version = "2.1.2" %}
-{% set sha256 = "667ed6019bb4e2f925bd9291161d2c61cc0077443094437b703ea905333fd585" %}
+{% set number = 3 %}
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set scs_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
-  name: {{ name|lower }}
+  name: scs-split
   version: {{ version|replace('-', '.') }}    # 2.1.1-2 => 2.1.1.2
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  # url: https://pypi.io/packages/source/s/scs/scs-{{ version }}.tar.gz
+  # sha256: 667ed6019bb4e2f925bd9291161d2c61cc0077443094437b703ea905333fd585
+  git_url: https://github.com/bodono/scs-python.git
+  git_rev: 134358356a0232ad9f2bf4dd3886f5b522322175
+  git_depth: 1
 
 build:
-  number: 2
-  script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py==27 and (win or (not x86_64))]
+  number: {{ number }}
+  # GPU version for linux64 & win
+  skip: true  # [osx and cuda_compiler_version != "None"]
```

Comment:
I'm surprised this is required, doesn't the matrix only include `cuda_compiler_version` = `None` when it's `osx`?

Reply:
You're probably right, will try removing.